weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/schemaUtils.ts

```diff
@@ -639,7 +633,12 @@ export function filterRowsForNewDataset(
 
         const {[WEAVE_NAMESPACE]: weaveData, ...rest} = row;
         const filteredData = Object.fromEntries(
-          Object.entries(rest).filter(([key]) => targetFields.has(key))
+          Object.entries(rest).filter(([key]) =>
```

Comment:
I don't see where `filterRowsForNewDataset` is getting used?

Reply:
Oh you're right I ripped that out because we are doing it within `createProcessedRowsMap`, I will just delete this function